abandoned_battlefield/map.conf

```
@@ -0,0 +1,26 @@
+initial_stuff = default:cobble 40,shooter:pistol,default:pick_steel,default:torch 8
+author = HobbitPower
+name = Abandoned Battlefield
+h = 140
+rotation = z
+team.1.pos = (75,2,97)
+r = 115
+team.1 = red
+team.1.color = red
+team.2.pos = (-75,2,-97)
+team.2 = blue
+team.2.color = blue
+license = CC BY-SA 4.0
+chests.1.n = 25
+chests.1.from = (-66,1,79)
+chests.1.to = (-84,1,101)
+chests.2.n = 25
+chests.2.from = (-27,1,79)
+chests.2.to = (-45,1,101)
+chests.3.n = 25
+chests.3.from = (66,1,-79)
+chests.3.to = (84,1,-101)
+chests.4.n = 25
+chests.4.from = (27,1,-79)
+chests.4.to = (45,1,-101)
+hint = Find ores hidden on a secret place!
```

```suggestion
hint = Find hidden ores on a secret place!
```
One more trivial thing. This might sound better in English.